Source/FullScreenPokemon.ts

```diff
@@ -9384,6 +9385,7 @@ module FullScreenPokemon {
          */
         checkForOldStorageData(): void {
             if (!this.ItemsHolder.getItem("oldLocalStorage") || this.ItemsHolder.getItem("gameStarted")) {
+                this.ItemsHolder.getItem("collectionKeys");
```

Comment:
Uhh nvm, I thought I commented this line out while testing a fix for this. I'm in the middle of porting `collectionKeys` to StateHoldr but if that is done, grabbing`collectionKeys` at the start will be better in StateHoldr.